docs/data/base/pagesApi.js

```diff
@@ -1,42 +1,169 @@
+    hash: 'popper-unstyled',
+    title: 'PopperUnstyled',
+  },
+  { pathname: '/base/react-portal/components-api', hash: 'portal', title: 'Portal' },
+  {
+    pathname: '/base/react-select/components-api',
+    hash: 'select-unstyled',
+    title: 'SelectUnstyled',
+  },
+  {
+    pathname: '/base/react-slider/components-api',
+    hash: 'slider-unstyled',
+    title: 'SliderUnstyled',
+  },
+  {
+    pathname: '/base/react-snackbar/components-api',
+    hash: 'snackbar-unstyled',
+    title: 'SnackbarUnstyled',
+  },
+  {
+    pathname: '/base/react-switch/components-api',
+    hash: 'switch-unstyled',
+    title: 'SwitchUnstyled',
+  },
+  {
+    pathname: '/base/react-tabs/components-api',
+    hash: 'tab-panel-unstyled',
+    title: 'TabPanelUnstyled',
+  },
+  {
+    pathname: '/base/react-tabs/components-api',
+    hash: 'tab-unstyled',
+    title: 'TabUnstyled',
+  },
+  {
+    pathname: '/base/react-table-pagination/components-api',
+    hash: 'table-pagination-unstyled',
+    title: 'TablePaginationUnstyled',
+  },
+  {
+    pathname: '/base/react-tabs/components-api',
+    hash: 'tabs-list-unstyled',
+    title: 'TabsListUnstyled',
+  },
+  {
+    pathname: '/base/react-tabs/components-api',
+    hash: 'tabs-unstyled',
+    title: 'TabsUnstyled',
+  },
+  {
+    pathname: '/base/react-textarea-autosize/components-api',
+    hash: 'textarea-autosize',
+    title: 'TextareaAutosize',
+  },
+  { pathname: '/base/react-badge/hooks-api', hash: 'use-badge', title: 'useBadge' },
+  {
+    pathname: '/base/react-button/hooks-api',
+    hash: 'use-button',
+    title: 'useButton',
+  },
+  {
+    pathname: '/base/react-form-control/hooks-api',
+    hash: 'use-form-control-unstyled-context',
+    title: 'useFormControlUnstyledContext',
+  },
+  { pathname: '/base/react-input/hooks-api', hash: 'use-input', title: 'useInput' },
+  { pathname: '/base/react-menu/hooks-api', hash: 'use-menu', title: 'useMenu' },
+  {
+    pathname: '/base/react-menu/hooks-api',
+    hash: 'use-menu-item',
+    title: 'useMenuItem',
+  },
+  {
+    pathname: '/base/react-select/hooks-api',
+    hash: 'use-option',
+    title: 'useOption',
+  },
+  {
+    pathname: '/base/react-select/hooks-api',
+    hash: 'use-select',
+    title: 'useSelect',
+  },
+  {
+    pathname: '/base/react-slider/hooks-api',
+    hash: 'use-slider',
+    title: 'useSlider',
+  },
+  {
+    pathname: '/base/react-snackbar/hooks-api',
+    hash: 'use-snackbar',
+    title: 'useSnackbar',
+  },
+  {
+    pathname: '/base/react-switch/hooks-api',
+    hash: 'use-switch',
+    title: 'useSwitch',
+  },
+  { pathname: '/base/react-tabs/hooks-api', hash: 'use-tab', title: 'useTab' },
+  {
+    pathname: '/base/react-tabs/hooks-api',
+    hash: 'use-tab-panel',
+    title: 'useTabPanel',
+  },
+  { pathname: '/base/react-tabs/hooks-api', hash: 'use-tabs', title: 'useTabs' },
+  {
+    pathname: '/base/react-tabs/hooks-api',
+    hash: 'use-tabs-list',
+    title: 'useTabsList',
+  },
```

Comment:
Something is off, we can see it 4 times

<img width="290" alt="Screenshot 2023-04-01 at 15 06 31" src="https://user-images.githubusercontent.com/3165635/229290855-43f34857-2b95-4476-8294-51528fa94955.png">

https://deploy-preview-35938--material-ui.netlify.app/base/react-tabs/hooks-api/#use-tab-panel

Reply:
Fixed in https://github.com/mui/material-ui/pull/36764